OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -109,6 +109,8 @@ public override Activity Tick(Actor self)
 			// Wait for the worm to get back underground
 			if (stance == AttackState.ReturningUnderground)
 			{
+				sandworm.IsAttacking = false;
+
 				// There is a 50-50 chance that the worm would just go away
 				if (self.World.SharedRandom.Next() % 2 == 0)
```

Comment:
Could you make that value an attribute of the worm manager and set it to 80% per default?